types/bootstrap/js/dist/dropdown.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as Popper from "@popperjs/core";
 import BaseComponent, { GetInstanceFactory, GetOrCreateInstanceFactory } from "./base-component";
+import Tooltip from "./tooltip";
```

Comment:
how this can depends on tooltip module? I assume one could extract common type out of tooltip for reuse